roles/tas_single_node/tasks/certificates.yml

```diff
@@ -77,11 +77,6 @@
     dns_name: "{{ cert_name }}.{{ tas_single_node_base_hostname }}"
 
 - name: Create Fulcio root
-  when: >
-    (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_fulcio_private_key) | list | length) == 0
-    or (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_fulcio_public_key) | list | length) == 0
-    or (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_fulcio_root_ca) | list | length) == 0
-
   block:
     - name: Create Fulcio private key
       # NOTE: As of cosign 2.3.0, only RSA with > 2048 B pkey size or ECC with prime256v1 (==secp256r1) are supported
```

Comment:
I think based on this comment we should document for users what type of key can be provided.